tutordiscovery/templates/discovery/build/discovery/Dockerfile

```diff
@@ -86,7 +86,10 @@ COPY --chown=app:app assets.py ./course_discovery/settings/assets.py
 RUN DJANGO_SETTINGS_MODULE=course_discovery.settings.assets make static
 
 # Create media directory to serve media files
+# have to change user at this step, as it gives permission error
+User app
```

Comment:
All previous steps are performed as user app, so we should not have to change user here. Also, we should not switch back to root user after creating the media root.

Reply:
Yes, you are right. For redwood, we are changing user to app at line 40. It shoulde work, i'll again test it on redwood.

But on quince,
I got below error message(I also added in the commit message) that's why I added this line. 
"""
discovery-1  | PermissionError: [Errno 13] Permission denied: '/openedx/discovery/course_discovery/media/media'
"""
I checked the ownership of the directory in bash, it was root.